src/screens/ResetPasswordScreen/ResetPasswordScreen.spec.tsx

```diff
@@ -11,6 +12,20 @@ import renderWithRedux from '~/utils/testing/renderWithRedux';
 import ResetPasswordScreen from './ResetPasswordScreen';
 
 jest.unmock('react-redux');
+// should it be moved into separated module?
+jest.mock('react-native', () => {
+  const RN = jest.requireActual('react-native');
+
+  return Object.setPrototypeOf(
```

Comment:
Can you explain a little such syntax? 😄 
Maybe returning `__esModule: true` in object would help? ([link](https://jestjs.io/docs/en/jest-object#jestrequireactualmodulename))